### DISCUSSION TITLE
Support for zstd compression

### DISCUSSION THREAD

**Discussion by harrisonhunter** [meteor/meteor#12347] [Feature Requests]
I have tried installing the @mongodb-js/zstd package, and it seems to install correctly. However, when I try to run my app with zstd in the compressors field of the query string. I get the following error:

errorClass [Error]: [An error occurred when creating an index for collection "users: Optional module '@mongodb-js/zstd' not found. Please install it to enable zstd compression]

Other compressors such as snappy work fine. I have tried this on both meteor 2.8.0 and 2.9.0.

Is there any guidance on getting zstd working with meteor or any common issues that might be preventing this from working?

**Comment by radekmie** [comment]
I saw your forum post and fully agree with that. I guess the problem here is that the driver is installed in Meteor [`npm-mongo` package's dependencies](https://github.com/meteor/meteor/blob/devel/packages/npm-mongo/package.js), and thus the `@mongodb-js/zstd` is not visible for it. I'm not sure how we could handle that on Meteor level, but probably @zodern can help here.
